Sidney-/slugtrade
# Js variable names

New changes:
- Renames variables with underscores to conform to JS naming conventions (camel case) in main.js

How to test:
- Make sure that the edit profile page still works (specifically the logic where an alert pops up when you try to leave the page after making changes)
- Make sure that the add closet items page still works (specifically the photo adding system {sorry about this!})
